charts/kubewarden-controller/Chart.yaml

```diff
@@ -21,9 +21,9 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.6.2
+version: 1.7.0-rc2
```

Comment:
This should not be `1.7.0-rc2`, but `2.0.0-rc1`. This is the `version` of the chart, and we have broken compatibility for the first time.

Reply:
Done in https://github.com/kubewarden/helm-charts/pull/333/commits/ff8ce16f5954a6e23be435922548d7000914185b.